auto_submit/lib/validations/revert.dart

```diff
@@ -102,10 +102,13 @@ class Revert extends Validation {
     if (bodyText == null) {
       return null;
     }
-    final RegExp regExp = RegExp(r'^[Rr]everts[\s]+([-\.a-zA-Z_]+/[-\.a-zA-Z_]+#[0-9]+)$', multiLine: true);
```

Comment:
Maybe we can just remove `$` to allow for additional comments but still requiring to start with the revert line?

Reply:
The dollar actually does end the string and not the line in this case. You see from the test cases that we allow trailing text now since there have been multiple instances where people want to add context to the pull request.